README.md

```diff
@@ -31,9 +31,8 @@ you can install the type stubs using
     $ pip install types-six types-requests
 
 These PyPI packages follow [PEP 561](http://www.python.org/dev/peps/pep-0561/)
-and are automatically generated by typeshed internal machinery. Also starting
-from version 0.900 mypy will provide an option to automatically install missing
-type stub packages (if found on PyPI).
+and are automatically released (multiple times a day, when needed) by
+[typeshed internal machinery](https://github.com/typeshed-internal/stub_uploader).
 
 PyCharm, pytype etc. work in a similar way, for more details see documentation
 for the type-checking tool you are using.
```

Comment:
Now this paragraph doesn't make sense, because it meant "similar to mypy". I would rewrite it as "Type checkers should be able to use these stub packages when installed. For more details, see the documentation for your type checker."